vipps-order-management-api.md

```diff
@@ -182,7 +182,7 @@ Example request:
 
 Headers:
 
-```text
+```json
```

Comment:
try using http instead

Reply:
Also, below there is a missing : after Vipps-System-Plugin-Version

Reply:
Thank you!!